### PR TITLE
Fix routes and index action for /systems/trust_relationships

### PR DESCRIPTION
**Pull request by knutsenm**
1) make index action actually produce output, and
2) make index action reachable by moving systems namespace before systems resources

Without the latter, /systems/trust_relationships yields "This isn't a uuid: trust_relationships".
That is, it's routed to the SystemsController.

### CODE REVIEW

**Comment by knutsenm** [comment]
@twinge, I noticed this while reviewing the code in preparation for adding my own new model, etc. for GR-37, "Tie api_keys (systems) to an ip_address"

File: config/routes.rb [hunk 1]
```diff
@@ -8,16 +8,16 @@
     end
   end
 
+  namespace :systems do
+    resources :trust_relationships, except: [:new, :edit, :show, :update]
+  end
+
```
Comment (twinge) [inline config/routes.rb]: Make sure that didn't obscure the other systems endpoint.

File: spec/routing/systems_trust_relationships_routing_spec.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+describe 'routes for Systems::TrustRelationships' do
+  it 'routes GET /systems/trust_relationships to the trust_relationships controller #index action' do
+    expect(get('/systems/trust_relationships')).to route_to('systems/trust_relationships#index')
```
Comment (knutsenm) [inline spec/routing/systems_trust_relationships_routing_spec.rb]: Without the routes.rb patch, this test fails with

```patch
--- expected
+++ actual
@@ -1 +1 @@
-{"controller"=>"systems/trust_relationships", "action"=>"index"}
+{"action"=>"show", "controller"=>"systems", "id"=>"trust_relationships"}
```

**Comment by twinge** [comment]
:+1:

**Pull request merged**